### PR TITLE
Write RSS to hardcoded path and move RSS workflow into main scraping workflow

### PR DESCRIPTION
**Pull request by jsvine**
- Write RSS to hardcoded path: Ultimately, I'm thinking we'll want to use this script to write many RSS feeds — e.g., one per state — so writing to `sys.stdout` will have limited future usefulness. 
- Move RSS workflow into main scraping workflow: Just makes things simpler, and guarantees that the discovery-dates and RSS generation happens immediately after the latest fetched data are committed.

### CODE REVIEW

**Review by m-nolan — APPROVED** [review]
As long as we're using the default RSS publishing length, this looks fine. Is the stdout change necessary if we can specify the output file in the .yaml files anyways? That might even allow for some code reuse later on. Maybe it's smarter to refactor the data scrubbing to something that can be iterated over states/regions/subsets and keep the hard-coded paths in the script (which seems to be what you're building to)

**Comment by jsvine** [comment]
Thanks, and I think you're right: It might be better to have both options — i.e., to *also* have an option to write out to `stdout`. Perhaps this can be handled through an additional CLI argument. But I'd say let's hold off on that until we've merged this into the main repo and smoothed out whatever wrinkles there need smoothing.

**Comment by m-nolan** [comment]
Great, makes sense. Should I do a PR to merge all of this to the main repo?

**Pull request merged**
